optaplanner-core/src/test/java/org/optaplanner/core/impl/heuristic/selector/value/chained/DefaultSubChainSelectorTest.java

```diff
@@ -162,12 +215,12 @@ public void original() {
         when(stepScopeB1.getPhaseScope()).thenReturn(phaseScopeB);
         subChainSelector.stepStarted(stepScopeB1);
         assertAllCodesOfSubChainSelector(subChainSelector,
-                "[a1]", "[a1, a2]", "[a1, a2, a4]",
-                "[a2]", "[a2, a4]",
-                "[a4]",
-                "[b1]", "[b1, a3]", "[b1, a3, b2]",
-                "[a3]", "[a3, b2]",
-                "[b2]");
+                                         "[a1]", "[a1, a2]", "[a1, a2, a4]",
```

Comment:
Code style: these should be aligned 2 idents in, not with the ( column.
The kie code style has been changed some time ago to align with Eclipse's limitations (which almost no one every uses), but that wasn't a good thing. We should restore the old one...

Reply:
BTW, on the long term I want to shortcut this discussion and consider just using the Quarkus code style. That one is Sun Code Style compliant, JBoss code style compliant and they enforce it in their build automatically - which really just erases all discussion.

Reply:
We should do it rather sooner than later - otherwise we will be discussing the indentation despite everybody having the KIE code style imported.

Reply:
Thanks for the feedback. Does this rule apply to other automatically indented lines as well? (e.g. starting at line 441)

Reply:
yes